README.Docker.md

```diff
@@ -0,0 +1,246 @@
+# ClamAV in Docker
+ClamAV can be run within a Docker container. This provides isolation from other
+processes by running it in a containerized environment. If new or unfamiliar
+with docker, containers or cgroups see [docker.com](https://www.docker.com).
+
```

Comment:
Can we remove the extra newline to match the style of our other markdown?

Reply:
ok